dataset/Femnist_stats.md

```diff
@@ -0,0 +1,91 @@
+
+# Understand the Heterogeneous FL Data [[Jupyter Notebook](/dataset/Femnist_stats.ipynb) ]
+
+
+## Download the Femnist dataset and FedScale
+Follow the download instruction in `./dataset/download.sh` to download the FEMNIST dataset.
```

Comment:
./ means it's in the current directory. Also, download instructions isn't in the download.sh file; they are in the README in the dataset directory. Include a link to it.